pkg/local/local_test.go

```diff
@@ -21,7 +21,26 @@ func TestCopyConfig(t *testing.T) {
 	}
 	d := &mockDestination{}
 
-	copied, err := CopyConfig("service-a", s, d)
+	copied, err := CopyConfig("service-a", s, d, "")
+	if err != nil {
+		t.Fatal(err)
+	}
+	d.assertFilesWritten(t, copiedfiles)
+	if !reflect.DeepEqual(copiedfiles, copied) {
+		t.Fatalf("failed to copy the files, got %#v, want %#v", copied, files)
+	}
+}
+
+func TestCopyConfigCanUseTargetDirectoryOverride(t *testing.T) {
+	s := &mockSource{localPath: "/tmp/testing"}
+	files := []string{"config/my-file.yaml", "config/this-file.yaml"}
+	copiedfiles := []string{"/environments/staging/services/service-a/base/config/my-file.yaml", "/environments/staging/services/service-a/base/config/this-file.yaml"}
+	for _, f := range files {
+		s.addFile(f)
+	}
+	d := &mockDestination{}
+
+	copied, err := CopyConfig("service-a", s, d, "/environments/staging")
```

Comment:
Is this an example of what I asked above in `pathForServiceConfig()`, why someone would do it?

Reply:
Yeah possibly, so we could just use `staging` here then look at fixing the actual problem, I'll do that